test/router.test.js

```diff
@@ -251,6 +251,7 @@ describe('Router', () => {
 		await sleep(10);
 
 		expect(scratch).to.have.property('innerHTML', '<h1>B</h1><p>hello</p>');
+		expect(B).to.have.been.calledOnce;
```

Comment:
Prior to this, `B` (or any async route) would be called twice after route change.